### PR TITLE
ci: deploy green pool ui in prod

### PR DESCRIPTION
**Pull request by royki**

### CODE REVIEW

File: .github/workflows/cf-deploy.yml [hunk 1]
```diff
@@ -23,8 +23,8 @@ jobs:
             echo "::set-output name=env_name::green-dev"
             echo "::set-output name=env_file::src/environments/environment.dev.ts"
             echo "::set-output name=build_script::npm run build"
-          elif [ "${{ github.ref }}" = "refs/heads/release" ]; then
-            echo "::set-output name=env_name::prod"
+          elif [ "${{ github.ref }}" = "refs/heads/green-release" ]; then
+            echo "::set-output name=env_name::green-release"
```
Comment (jrhender) [inline .github/workflows/cf-deploy.yml:27]: @dawidgil is this the right `env_name`?

Comment (jrhender) [reply .github/workflows/cf-deploy.yml:27]: Actually, I think this is okay. I think this just configures the GitHub environment name, as far as I can tell

**Review by jrhender — APPROVED** [review]

**Review by jrhender — CHANGES_REQUESTED** [review]
Need to confirm if CF access is applied

File: .github/workflows/cf-deploy.yml (hunk 1 above)
Comment (dawidgil) [reply .github/workflows/cf-deploy.yml:27]: Yeah, this is Github env stuff.

**Review by jrhender — APPROVED** [review]
CF applied

**Pull request merged**